carousel-backend/routes/routes.js

```diff
@@ -0,0 +1,17 @@
+
+const express = require('express')
+const router = express.Router()
+const {sharksList,catsList} = require('../mock/input')
+
+
+router.get('/cats', (req,res) => {
+    res.status(200).send(catsList)
+})
+
+router.get('/sharks',(req,res) => {
+    res.status(200).send(sharksList)
+})
```

Comment:
What types of changes could you make to this API design to make it "more RESTful"?

Reply:
1. First in order to make it RESTful I would setup CRUD(create,read,update, delete) routes(assuming we are connected to a database to store content) for cats and sharks which will involve GET, POST, PUT/PATCH, DELETE calls. I can do it two ways.  I can build a CRUD route and use either req.query or req.params value as a key to decide which content to be provided or build seperate CRUD for cats and seperate CRUD for sharks.
2. Build Eventemitters to emit output responses(responses in text as well as response codes).
3. I will setup middlewares(JWT checks, caching) to check if the incoming request contains proper headers. 
4. I will setup Joi validation to validate the incoming request body matches in terms of types and fields.
5. I will provide filtering, sorting mechanism for each routes based on values in request query.